zapcore/write_syncer.go

```diff
@@ -75,6 +78,99 @@ func (s *lockedWriteSyncer) Sync() error {
 	return err
 }
 
+type bufferWriterSyncer struct {
+	sync.Mutex
```

Comment:
Style: Please add an empty line between embedded fields and regular fields.

```suggestion
	sync.Mutex

```

https://github.com/uber-go/guide/blob/master/style.md#embedding-in-structs

Reply:
unresolved, since this doesn't look done